prac_08/dynamic_labels.py

```diff
@@ -0,0 +1,33 @@
+"""
+CP1404 Prac 8
+Dynamic labels exercise
+"""
+
+from kivy.app import App
+from kivy.lang import Builder
+from kivy.uix.label import Label
+
+
+class DynamicLabelsApp(App):
```

Comment:
Well done! The code is clean and precise. Docstring and naming are appropriate as well.